WorkflowReactiveSwift/Sources/Worker.swift

```diff
@@ -24,19 +26,51 @@ import ReactiveSwift
 /// If there is, and if the workers are 'equivalent', the context leaves the existing worker running.
 ///
 /// If there is not an existing worker of this type, the context will kick off the new worker (via `run`).
-public protocol Worker {
+public protocol Worker: AnyWorkflowConvertible {
```

Comment:
Nit: Should the conformance be moved to the extension below?

Reply:
Since `Worker` is a protocol, we get the `Extension of a protocol cannot have an inheritance clause` error.

Reply:
Ah. Yes. 👍